web/components/Feed/Feed.tsx

```diff
@@ -128,7 +133,7 @@ const filterGroups: FilterGroup[] = [
           { value: Breed.Whippet, label: "Whippet" },
         ],
         dropdown: true,
-        allSelected: true,
+        allSelected: false,
```

Comment:
Keep this as `true` - even though the filter is "Breed Restrictions" I think it acts as an _inclusive_ filter rather than an _exclusive_ one.

Reply:
Disregard this - just read the note written by PM. Approving!